torch/csrc/autograd/function.h

```diff
@@ -699,7 +699,7 @@ template <typename... Variables>
 edge_list collect_next_edges(Variables&&... variables) {
   detail::MakeNextFunctionList make;
   make.apply(std::forward<Variables>(variables)...);
-  return std::move(make.next_edges);
```

Comment:
What's wrong with this one? Without the move it would be a copy, is the next_edges attribute trivially copyable?

Reply:
I thought it may prevent copy elision in c++17, but now I am sure it is not after experiments, I will revert it.